core/history_api.php

```diff
@@ -238,6 +285,31 @@ function history_get_range_result_filter( $p_filter, $p_start_time = null, $p_en
 }
 
 /**
+ * Creates and executes a query for the history rows related to bugs matched by the provided filter
+ * @param  array $p_filter           Filter array
+ * @param  integer $p_start_time     The start time to filter by, or null for all.
+ * @param  integer $p_end_time       The end time to filter by, or null for all.
+ * @param  string  $p_history_order  The sort order.
+ * @return database result to pass into history_get_event_from_row().
+ */
+function history_get_range_result_filter( $p_filter, $p_start_time = null, $p_end_time = null, $p_history_order = null ) {
```

Comment:
This function could be removed as it was introduced recently only to allow timeline on recursive subprojects, and has been replaced by history_query_result()